actions/src/sema4ai/actions/agent/_client.py

```diff
@@ -0,0 +1,196 @@
+import json
+import logging
+import os
+import platform
+from copy import copy
+from urllib.parse import urljoin, urlparse
+
+import sema4ai_http
+
+log = logging.getLogger(__name__)
+
+
+class _AgentAPIClient:
+    API_ENDPOINT_VERSIONS = ["v1", "v2"]
+    API_PATH_PREFIX = "api/public"
+    ENV_VAR_NAME = "SEMA4AI_AGENTS_SERVICE_URL"
```

Comment:
`ENV_VAR_NAME` doesn't sem a good name for a variable... maybe it could be more descriptive (say `SEMA4AI_AGENT_SERVICE_URL_ENV_VAR_NAME`).

As a note, if it's used just in one place, maybe I wouldn't even create a separate constant...